x-pack/plugins/maps/public/connected_components/mb_map/utils.ts

```diff
@@ -73,15 +80,14 @@ function isCrossOriginUrl(url) {
   );
 }
 
-export async function loadSpriteSheetImageData(imgUrl) {
+export async function loadSpriteSheetImageData(imgUrl: string): Promise<ImageData> {
   return new Promise((resolve, reject) => {
     const image = new Image();
     if (isCrossOriginUrl(imgUrl)) {
       image.crossOrigin = 'Anonymous';
     }
-    image.onload = (el) => {
-      const imgData = getImageData(el.currentTarget);
```

Comment:
el.currentTarget just returned "image", so removed the unneeded complexity. Plus el.currentTarget is typed as "EventTarget | null" so then there was a weird edge case that needed to be handled.

Another thing for clarity is "el" should actually be "event" since its an Event and not an Element